extensions-contrib/distinctcount/README.md

```diff
@@ -0,0 +1,51 @@
+Discuss at https://groups.google.com/forum/#!topic/druid-development/GXRpXfBzfJs
+(1) First use https://github.com/druid-io/druid/pull/2570 to partition data by a dimension for example visitor_id.
```

Comment:
also pls indicate the assumption that one value has to be present in only one segment or this might overcount.